src/main/java/ch/uzh/ifi/hase/soprafs21/service/UserService.java

```diff
@@ -39,4 +46,39 @@ public User createUser(User newUser) {
         return null;
     }
 
+    /**
+     * This is a helper method that will check the uniqueness criteria of the username and the name
+     * defined in the User entity. The method will do nothing if the input is unique and throw an error otherwise.
+     *
+     * @param userToCheck
+     * @throws org.springframework.web.server.ResponseStatusException
+     * @see User
+     */
+    private void checkIfUserExists(User userToCheck) {
```

Comment:
since I have implemeted this method also, it may be good to delete it here to avoid merge conflicts ?

Reply:
Would be possible, however there will probably be quite a few merge conflicts (also with #67) because of incomplete implementation of the `User` class as well...